metasynth/__main__.py

```diff
@@ -68,7 +98,7 @@ def main():
     if args.num_rows is not None:
         data_frame = meta_frame.synthesize(args.num_rows)
     else:
-        data_frame = meta_frame.synthesize(meta_frame.n_rows)
+        data_frame = meta_frame.synthesize(meta_frame.n_rows)  # type: ignore [arg-type]
```

Comment:
Why is here an ignored type check? With the latest version, should it not accept None values?